SmartDeviceLink/SDLPermissionManager.h

```diff
@@ -18,6 +18,12 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface SDLPermissionManager : NSObject
 
+@property (strong, nonatomic, readonly) NSMutableDictionary<SDLPermissionRPCName, SDLPermissionItem *> *permissions;
+
+@property (assign, nonatomic, readonly) BOOL requiresEncryption;
```

Comment:
Documentation is needed for all public methods and properties. What does this do and why is it here?